### PR TITLE
Ft update user profile 

### PR DESCRIPTION
**Pull request by Brendajoshua**
# Description

Add attribution link to Readme and code

## Type of change

Please delete options that are not relevant.
- [x] New feature (non-breaking change which adds functionality)

## Change status

- [ ] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

Please describe steps taken to ensure this feature is functional and does not break other features:

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation, if relevant
- [x] I have run the app using my feature and ensured that no functionality is broken
- [x] My changes generate no new warnings
- [x] There are no merge conflicts

### CODE REVIEW

File: api/controllers/user.js [hunk 1]
```diff
@@ -65,5 +65,21 @@ module.exports = {
...
+  },
+  async updateUserProfile(req, res, next) {
+    const { userId } = req.params;
+    const {profilePicture, Location} = req.body;
+    try {
+      const updateUser = await models.Users.update (
+        {
+          profile_picture: profilePicture,
+          location: Location
+        },
```
Comment (iamcristos) [inline api/controllers/user.js]: this syntax looks like knex with PostgreSQL are you sure it works on mongoose?

File: api/routes/authroute.js [hunk 2]
```diff
@@ -8,6 +8,6 @@ const router = express.Router();
 router.post("/signup",[userValidators.validateUserOnSignup], controller.createUser);
 router.post("/login", controller.loginUser);
 router.patch("/confirm", controller.activateUser);
-
+router.patch("/updateprofile", controller.updateUserProfile);
```
Comment (iamcristos) [inline api/routes/authroute.js]: you are not passing the Id are params here

File: api/controllers/user.js [hunk 3]
```diff
@@ -80,13 +81,45 @@ module.exports = {
   },
   async updateProfile(req, res, next) {
     try{
-      const updatedUser = await models.findByIdAndUpdate({ userId: req.body.userId }).exec();
-      if (updatedUser) {
-        return {updatedUser, userId: updatedUser.id };
+      console.log(req.user);
```
Comment (OloruntobiAwoderu) [inline api/controllers/user.js]: You should remove this console.log statement

Comment (ruairidhflint) [reply api/controllers/user.js]: ^^ Agreed.

Comment (Brendajoshua) [reply api/controllers/user.js]: Done, thank you.

**Pull request merged**